source/extensions/filters/network/http_connection_manager/config.h

```diff
@@ -179,6 +180,23 @@ class HttpConnectionManagerConfig : Logger::Loggable<Logger::Id::config>,
   }
   std::chrono::milliseconds delayedCloseTimeout() const override { return delayed_close_timeout_; }
   const LocalReply::LocalReply& localReply() const override { return *local_reply_; }
+  void normalizePath(Http::RequestHeaderMap& request_headers) const override {
```

Comment:
This may not be the best place for this method. `HttpConnectionManagerConfig` mostly keeps immutable configuration values. I would suggest moving this method into `ConnectionManagerUtility` and just adding accessors to the `forwarding_path_transformer_` and `filter_path_transformer_` in here.

Reply:
Thanks! Will clean it up.